test/support.js

```diff
@@ -84,6 +84,10 @@ givenFile = function(name, pathToFile) {
   };
 };
 
+initializePackage = function() {
```

Comment:
Since this helper is used only by `installSandboxPackages`, please move its definition the `test/end-to-end.js`.